cmake/Modules/FindExprTk.cmake

```diff
@@ -0,0 +1,7 @@
+include(${PROJECT_SOURCE_DIR}/nuclear/cmake/Modules/HeaderLibrary.cmake)
+
+HeaderLibrary(
+  NAME ExprTk
```

Comment:
Isn't this library in our codebase somewhere? we should probably delete it while doing this